src/modules/integrator-guide/pages/api-gw/javascript-processing.adoc

```diff
@@ -109,7 +109,6 @@ return {
 };
 ----
 
-
```

Comment:
Try not to leave repeated blank lines; it usually changes nothing and is sort of a programmer quirk

Reply:
I'll find another way to keep up with what files I've been through.